incubator/elasticsearch/values.yaml

```diff
@@ -28,8 +32,10 @@ client:
 
 master:
   name: master
-  replicas: 2
+  replicas: 3
   heapSize: "128m"
+  storage: "10Gi"
```

Comment:
Recommend dropping this to 4G. Smallest EBS volume size and the metadata on the master is very little data.